### ISSUE TITLE
Vagrant native --debug flag showing command help

### ISSUE THREAD

**Issue by brgnepal** [projectatomic/vagrant-service-manager#206]
Vagrant native `--debug` flag is showing help.

```
$ bundle exec vagrant service-manager env --debug  
 INFO global: Vagrant version: 1.8.2.dev
 INFO global: Ruby version: 2.2.1
 INFO global: RubyGems version: 2.4.8
 INFO global: VAGRANT_INTERNAL_BUNDLERIZED="1"
 INFO global: VAGRANT_LOG="debug"
 INFO global: Plugins:
 INFO global:   - bundler = 1.10.6
 INFO global:   - vagrant-registration = 1.2.1
 INFO global:   - vagrant-service-manager = 1.0.1
DEBUG global: Loading core plugin: /home/budhram/.rvm/gems/ruby-2.2.1/bundler/gems/vagrant-788bd557c4d5/plugins/communicators/winrm/plugin.rb
.................
 INFO cli: CLI: [] "service-manager" ["env"]
DEBUG cli: Invoking command class: Vagrant::ServiceManager::Command ["env"]
.................
DEBUG subprocess: Waiting for process to exit. Remaining to timeout: 32000
DEBUG subprocess: Exit status: 0
 INFO interface: info: Usage: vagrant service-manager env [object] [options]

Objects:
      docker      display information and environment variables for docker
      openshift   display information and environment variables for openshift

If OBJECT is ommitted, display the information for all active services

Options:
      --script-readable  display information in a script readable format.
      -h, --help         print this help
Usage: vagrant service-manager env [object] [options]

Objects:
      docker      display information and environment variables for docker
      openshift   display information and environment variables for openshift

If OBJECT is ommitted, display the information for all active services

Options:
      --script-readable  display information in a script readable format.
      -h, --help         print this help
 INFO environment: Running hook: environment_unload
 INFO host: Autodetecting host type for [#<Vagrant::Environment: /home/budhram/redhat/vagrant-service-manager>]
DEBUG host: Trying: slackware
DEBUG host: Trying: suse
DEBUG host: Trying: gentoo
DEBUG host: Trying: arch
DEBUG host: Trying: darwin
DEBUG host: Trying: freebsd
DEBUG host: Trying: redhat
 INFO host: Detected: redhat!
 INFO runner: Preparing hooks for middleware sequence...
 INFO runner: 1 hooks defined.
 INFO runner: Running action: environment_unload #<Vagrant::Action::Builder:0x000000023bbf58>
```

**Comment by hferentschik** [comment]
What's about introducing a proper logger into the code base. This would imo be helpful for development as well. In the case of '--debug' one could utilize the logger to log using the appropriate Vagrant channels.

**Comment by brgnepal** [comment]
@hferentschik We are usind Vagrant logger now. WIP in [this branch](https://github.com/budhrg/vagrant-service-manager/tree/fix-168)

**Comment by hferentschik** [comment]
>  We are usind Vagrant logger now

+1 that's what I was hoping for. Aligning with Log4r and Vagrants use of the logger.

**Comment by hferentschik** [comment]
So is this really a duplicate of #168?

**Comment by brgnepal** [comment]
The PR https://github.com/projectatomic/vagrant-service-manager/pull/217 fixes this issue too along with #168
